### DISCUSSION TITLE
Url for announcements are broken.

### DISCUSSION THREAD

**Discussion by Sour-abh-Raj** [alshedivat/al-folio#2123] [Q&A]
I have created a fork : https://github.com/Sour-abh-Raj/lucifer-dev. I have created a news announcement. The link to the announcement is correct but the announcement markdown file is not present at that location. Also links like [projects](/projects) in about.md are not redirecting to the correct url. i have specified my repo as my baseurl but it's not taking effect. I am also unable to get the 400, 800 and 1440 webp images.

**Comment by george-gca** [comment]
> The link to the announcement is correct but the announcement markdown file is not present at that location.

What do you mean by that?

> Also links like [projects](https://github.com/projects) in about.md are not redirecting to the correct url.

Check if you are pointing that right. In our [_pages/about.md](https://raw.githubusercontent.com/alshedivat/al-folio/master/_pages/about.md) we link to another section of the profile. This is done like this:

```markdown
[publications page](/al-folio/publications/)
```

> I am also unable to get the 400, 800 and 1440 webp images.

Which image format you are using? Because svgs do not need this kind of treatment since [svg is a vector image format](https://en.wikipedia.org/wiki/SVG).

**Reply by Sour-abh-Raj** [reply]
Sorry if I wasn't clear. So when you create a news announcement with inline set to false and click on the announcement it's supposed to serve that announcement markdown file as <url>\<news>\<path>. I created an announcement earlier with inline set to false but when I click on that announcement link I get 404 error.

**Reply by george-gca** [reply]
I am not sure that this is the problem, but checking your news file the dates is `date: 2024-01-25 12:35:00+0530` while in ours the format is a little different `date: 2015-11-07 16:11:00-0400`. Check if this small change fixes it.

**Reply by george-gca** [reply]
If this doesn't work, I'd suggest you check your changes from the original repo, see if any could have caused this issue.
